src/xcode/ENA/ENA/Resources/Localization/bg.lproj/Localizable.links.strings

```diff
@@ -45,3 +45,5 @@
 "CovPass_Check_FAQLink" = "https://www.coronawarn.app/en/faq/#eu_dcc_check";
 
 "NotificationSettings_FAQLink" = "https://www.coronawarn.app/en/faq/#notification_settings";
+
+"TicketValidation_FAQLink" = "https://www.coronawarn.app/en/faq/#val_service_result";
```

Comment:
Since we have other FAQ links for ticket validation, I would suggest to make it more specific. 

```suggestion
"TicketValidation_ServiceResult_FAQLink" = "https://www.coronawarn.app/en/faq/#val_service_result";
```

Reply:
Ah, I wasn't aware that we have multiple FAQ links for this, renamed it now 👍